spec/services/dialog_field_visibility_service_spec.rb

```diff
@@ -92,7 +92,7 @@
 
       allow(number_of_vms_visibility_service)
         .to receive(:determine_visibility).with(number_of_vms, platform).and_return(
-          :hide => [:number_hide], :edit => [:number_edit]
+          :hide => [:number_hide, :floating_ip_address], :edit => [:number_edit, :floating_ip_address]
```

Comment:
You don't have to do this here, it's just a canned response so changing the response doesn't really buy us anything. Notice we're not including the `:sysprep_computer_name` or `:linux_host_name` either.